### DISCUSSION TITLE
the new pipeline log print is less than old print

### DISCUSSION THREAD

**Discussion by ljl20032021789** [opendilab/DI-engine#723] [Q&A] [answered]
hi  ,
  the new pipeline generated files do not include log director, collector director , evaluator director, and learner director.
  the tensorboard print information is less than old pipeline
  how to  open it in new pipeline?

   the new pipeline print is less then old pipeline.
   there are no  print as follow :
   [09-14 17:04:05] INFO     collect end:                                                                                                      sample_serial_collector.py:406
                          episode_count: 26                                                                                                                               
                          envstep_count: 5120                                                                                                                             
                          train_sample_count: 5120                                                                                                                        
                          avg_envstep_per_episode: 196.92307692307693                                                                                                     
                          avg_sample_per_episode: 196.92307692307693                                                                                                      
                          avg_envstep_per_sec: 1007.5831581485946                                                                                                         
                          avg_train_sample_per_sec: 1007.5831581485946                                                                                                    
                          avg_episode_per_sec: 5.116633224973333                                                                                                          
                          reward_mean: 196.92307692307693                                                                                                                 
                          reward_std: 15.384615384615381                                                                                                                  
                          reward_max: 200.0                                                                                                                               
                          reward_min: 120.0                                                                                                                               
                          total_envstep_count: 18542                                                                                                                      
                          total_train_sample_count: 18542                                                                                                                 
                          total_episode_count: 223                                                                                                                        
[09-14 17:04:10] INFO     [RANK0]: === Training Iteration 500 Result ===

**Comment by PaParaZz1** [comment, marked as answer]
We will working on this issue on #735. You can keep track of it.
